src/essentials/Colors/types.ts

```diff
@@ -9,11 +9,12 @@ export type SemanticColorsSchema = {
     white: Color;
     black: Color;
     background: {
+        default: Color;
         page: {
-            default: Color;
-            'elevation-1': Color;
-            'elevation-2': Color;
-            'elevation-3': Color;
+            default: Color; // deprecated - use background.default or surface.default instead
```

Comment:
In alignment with Artur's comments, I also understood after the call with design that the only changes to our semantic tokens would be adding `elevation-0`

Reply:
Yup, you're right Marti. I applied the changes in 86d8c03.